src/test/java/org/apache/accumulo/proxy/its/SimpleProxyBase.java

```diff
@@ -2726,6 +2727,7 @@ public void testGetRowRange() throws Exception {
   }
 
   @Test
+  @Ignore
```

Comment:
Should probably add a TODO or other comment pointing to an open issue about its failure.

Reply:
Actually, I think I've figured out the problem. The test relies on a test jar, which is in Accumulo, and not in the proxy repository. A quick fix is probably to copy the test jar into the test resources directory for the proxy. I've already done this and am building it now, so I'll push that fix momentarily (once I've confirmed that it fixes the problem).

Reply:
I pushed a fix to this test just now (099e6863123e6de72aa5c4439c9c033bf129c494), so this PR isn't needed any more.